.travisci/codecov.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mvn jacoco:report
+$HOME/.script/codecov.bash -F"unit$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/powerunit/powerunit-extensions-matchers/pullRequest?prid=5559528)